articles/references/toc.yml

```diff
@@ -0,0 +1,13 @@
+- name: Concepts
+  href: concepts/toc.yml
+  topicHref: concepts/index.md
+- name: Module Builder
+  href: module_builder.md
+- name: Templates & Decorators
+  href: templates_decorators/toc.yml
+  topicHref: templates_decorators/index.md
+- name: Modules
+  href: modules/toc.yml
+  topicHref: modules/index.md
+- name: Core Modules
+  href: core_modules/toc.yml
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.